docs/modules/ROOT/pages/usage.adoc

```diff
@@ -85,8 +85,7 @@ The cluster definition also supports overriding configuration properties and env
 IMPORTANT: Do not override port numbers. This will lead to cluster malfunction.
 
 === Configuration Properties
-
-Currently, not supported for any file.
+You can override configuration properties by using configOverrides in your service configuration as demonstrated in the xref:config_properties.adoc[configuration properties] docs.
```

Comment:
I looked at the code and its only working for bootstrap.conf and nifi.properties. Maybe its worth mentioning that?

Reply:
I've only given examples for overriding bootstrap.conf and nifi.properties in the docs, but being explicit about this limitation does make sense.